packages/network/lib/cors.ts

```diff
@@ -106,11 +203,27 @@ export function urlMatchesOriginProtectionSpace (urlStr, origin) {
   return _.startsWith(normalizedUrl, normalizedOrigin)
 }
 
-export function getOriginPolicy (url: string) {
+export function getOrigin (url: string) {
   // @ts-ignore
-  const { port, protocol } = new URL(url)
+  const { port, protocol, hostname } = new URL(url)
 
   // origin policy is comprised of:
+  // protocol + subdomain + superdomain + port
+  return _.compact([`${protocol}//${hostname}`, port]).join(':')
```

Comment:
Could this be URL.origin now? https://developer.mozilla.org/en-US/docs/Web/API/URL/origin

Reply:
looks that way according to the unit tests. We will see if other tests fail, but this is addressed in eafb617e62